site/docs/v1/tech/5-minute-setup-guide.adoc

```diff
@@ -6,34 +6,62 @@ description: This guide will walk you through the steps required to setup Fusion
 
 :sectnumlevels: 0
 
-This is a 5-minute guide to to setup and integrate with FusionAuth. Starting... now!
+This is a 5-minute guide to set up and integrate with FusionAuth. Starting... now!
+
+At the end of this guide, you'll have a running Node application which will use FusionAuth for authentication, authorization and user management.
 
 == Requirements
 
-This guide assumes that you have installed either a MySQL or PostgreSQL database and have a super username and password for the database that is able to create schemas and databases. If you do not have a database installed, follow the instructions in our link:/docs/v1/tech/installation-guide/database/[Database Installation Guide] to install a database.
+This guide assumes that you have installed either a MySQL or PostgreSQL database and have a superuser name and password for the database that is able to create schemas and databases. If you do not have a database installed, follow the instructions in our link:/docs/v1/tech/installation-guide/database/[Database Installation Guide] to install a database.
+
+You also need to have Node installed. This code has been tested with node 14, but should work with other versions.
+
+[source,shell script,title=Testing Node Installation]
+----
+node -v
+----
+
+[source,shell script,title=Testing Node Installation Output]
+----
+v14.4.0
+----
+
+[source,shell script,title=Testing Postgresql Installation]
+----
+psql --version
+----
+
+[source,shell script,title=Testing Postgresql Installation Output]
+----
+psql (PostgreSQL) 12.3
+----
+
+
 
 == Overview
 
 Here are the steps that we need to take to setup FusionAuth as a login provider (also known as an identity provider) for your application:
 
 . Install FusionAuth
 . Start FusionAuth
-. Complete Maintenance Mode and the Setup Wizard
+. Complete Maintenance Mode 
+. Complete the Setup Wizard
 . Create an Application and configure the OAuth settings
 . Copy the OAuth login URL and add it to your application
 . Grant the user permissions to your application
 . Write a route (controller, action, etc) in your backend application that completes the login
 . Store the user object in the session
+. Test the application
```

Comment:
We can make these links by just changing them to `<<Install FusionAuth>>` (for example)

Reply:
Made the links work.